internal/cloudapi/v2/openapi.v2.yml

```diff
@@ -669,6 +677,10 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/User'
+        repositories:
```

Comment:
Hm can we add a description detailing that this is about payload/extra repositories? Now there will be two places where you can specify repos in the request and it should be obvious that these are non-essential. Optionally rename to `payload_repositories` or something, wdyt @teg naming is hard .-.